crates/compiler/builtins/bitcode/src/str.zig

```diff
@@ -2315,7 +2324,6 @@ test "strTrim: large to large" {
 test "strTrim: large to small" {
     const original_bytes = "             hello         ";
     const original = RocStr.init(original_bytes, original_bytes.len);
-    defer original.deinit();
```

Comment:
Why is this defer removed? It should still succeed since `original` remains unique, no?

Reply:
original is passed to `strTrim`, which will consume it. (so doing the deinit here would double-free)

Reply:
oh oops I see now `small_or_shared` is based on the resulting length. my bad